LongWarOfTheChosen/Config/XComAI.ini

```diff
@@ -155,6 +174,92 @@
 +AoEProfiles=( Profile=FlameThrowerIgnoreFF, Ability=AdvPurifierFlamethrower, bIgnoreSelfDamage=True, bFailOnObjectiveFire=true, bFailOnFriendlyFire=false, MinTargets=1, bAreaSearchSpace=1, bRequireLoS=1, bCountAdjacentAsHits=1, bTestTargetEffectsApply=true, bIgnoreRepeatAttackList=1,)
 +AoEProfiles=( Profile=FlameThrowerMimicBeacon, Ability=AdvPurifierFlamethrower, bIgnoreSelfDamage=True, bFailOnObjectiveFire=true, bFailOnFriendlyFire=false, MinTargets=1, bTargetEnemy=0, bAreaSearchSpace=1, bRequireLoS=1, bRequirePotentialTarget=1, bTestTargetEffectsApply=true, bIgnoreRepeatAttackList=1)
 +AoEProfiles=( Profile=FlameThrowerMimicBeaconAdjacent, Ability=AdvPurifierFlamethrower, bIgnoreSelfDamage=True, bFailOnObjectiveFire=true, bFailOnFriendlyFire=false, MinTargets=1, bTargetEnemy=0, bAreaSearchSpace=1, bRequireLoS=1, bCountAdjacentAsHits=1, bRequirePotentialTarget=1, bTestTargetEffectsApply=true, bIgnoreRepeatAttackList=1)
+
+; ----------------------------------------------------------------------------------------------------------
+; --------------------------- ADVENT Priest --------------------------------------------------
+; ----------------------------------------------------------------------------------------------------------
++NewBehaviors=(BehaviorName=PriestRedLastActionSelector, NodeType=Selector, \\
+Child[0]=TryHighPriorityShot, \\
+Child[1]=TryStasis, \\
+Child[2]=TryPsiMindControl, \\
+Child[3]=MoveDefensiveWithLoS, \\
+Child[4]=HuntEnemyWithCover)
++NewBehaviors=(BehaviorName=PriestRedFirstActionSelector, NodeType=Selector, \\
+Child[0]=DoIfFlankedMove, \\
+Child[1]=TryStasis, \\
+Child[2]=TryHighPriorityShot, \\
+Child[3]=TryPsiMindControl, \\
+Child[4]=TryHolyWarrior, \\
+Child[5]=TryShootOrReloadOrOverwatch, \\
+Child[6]=HuntEnemyWithCover)
+
++NewBehaviors=(BehaviorName=PriestRedFirstAction, NodeType=Sequence, Child[0]=NotLastActionPoint, Child[1]=PriestRedFirstActionSelector)
++NewBehaviors=(BehaviorName=PriestRedLastAction, NodeType=Sequence, Child[0]=IsLastActionPoint, Child[1]=PriestRedLastActionSelector)
+
+
++BehaviorRemovals="SelectPriestAbility"
++NewBehaviors=(BehaviorName=SelectPriestAbility, NodeType=Selector, Child[0]=MimicBeaconBehavior, Child[1]=PriestRedFirstAction, Child[2]=PriestRedLastAction, Child[3]=AdventRedFallbackAction)
+
++NewBehaviors=(BehaviorName=TargetAffectedByEffect-TransferMecToOutpost, NodeType=Condition)
++NewBehaviors=(BehaviorName=TargetScoreByScaledDistance, NodeType=Action, Param[0]="1")
++NewBehaviors=(BehaviorName=ScoreTargetIfMindControlledPositive, NodeType=Sequence, Child[0]=TargetAffectedByEffect-MindControl, Child[1]=AddToTargetScore_500)
```

Comment:
So this makes mind-controlled allies of the Priest higher priorities, right?

Reply:
Yes